itest/overcast-docker-itest/overcast.conf

```diff
@@ -4,10 +4,30 @@
 //   dockerHost=http://localhost:2375
 // }
 
-dockerMinimalConfig {
+dockerMinimalConfig {   
   dockerImage="busybox"
   command=["/bin/sh", "-c", "while true; do echo hello world; sleep 1; done"]
   remove=true
+  removeVolume=true
+} 
+
+mountebankConfig {      
+  dockerHost="http://localhost:2375"
+  dockerHost=${?overcastItest.dockerHost}                                
+  dockerImage="expert360/mountebank:1.3.0"
+  name="mountebank"
+  remove=true
+  removeVolume=true
+}
+
```

Comment:
mountebankConfig provisions [mountebank](http://www.mbtest.org/), which will be accessed through the link